test/integration/ForkTest.t.sol

```diff
@@ -207,8 +298,8 @@ contract ForkTest is Test {
         vm.stopPrank();
     }
 
-    function transfer_DelegatorNode(address asset, uint256 amountToTransfer) public {
-        vm.prank(manager);
+    function transfer_DelegatorNode(address asset, uint256 amountToTransfer) internal {
+        vm.prank(Addresses.MANAGER_ROLE);
```

Comment:
I kind of like the shorter `manager` syntax better? We can set the local manager from the addresses.

Reply:
I wanted to be explicit about what was being used. Before the `manager` and `operator` variables were set to the multisig address which will not be the case after this change.